guide/index.md

```diff
@@ -6,81 +6,218 @@ sidebar_position: 1
+          trackId={track.id}
+          videoLayout="FIT"
+        />
+      ))}
+    </View>
+  );
+}
 
-function Component() {
-  const { participants } = useParticipants();
+const styles = StyleSheet.create({
+  tracksContainer: {
+    flex: 1,
+    alignItems: "center",
+  },
+  videoElement: {
+    width: "85%",
+    aspectRatio: 1,
+    backgroundColor: "gray",
+    marginVertical: 16,
+  },
+});
+```
+
+### Full example
+
+Here is how it all could work together:
+
+:::info
+We are using expo-camera to request camera permissions. You can install and build it using the following command:
 
-  const videoTracks = participants.flatMap((participant) =>
-    participant.tracks.filter(
-      (track) => track.type === 'Video' && track.isActive,
-    ),
+```bash
+npx expo install expo-camera && npx expo prebuild
+```
+
+:::
+
+```tsx
+import { useCallback, useEffect } from "react";
+import { Button, StyleSheet, View } from "react-native";
+import {
+  joinRoom,
+  useCamera,
+  usePeers,
+  VideoRendererView,
+} from "@fishjam-cloud/react-native-client";
+import { SafeAreaView } from "react-native-safe-area-context";
+import { useCameraPermissions } from "expo-camera";
+
+function TracksView() {
+  const { peers } = usePeers();
+
+  const videoTracks = peers.flatMap((peer) =>
+    peer.tracks.filter((track) => track.type === "Video" && track.isActive),
   );
 
   return (
-    <View>
+    <View style={styles.tracksContainer}>
       {videoTracks.map((track) => (
-        <VideoRendererView key={track.id} trackId={track.id} videoLayout="FIT" />
+        <VideoRendererView
+          style={styles.videoElement}
+          key={track.id}
+          trackId={track.id}
+          videoLayout="FIT"
+        />
       ))}
     </View>
   );
 }
+
+function StartStreamingButton({ roomName, userName }) {
+  const { prepareCamera } = useCamera();
+
+  const startStreaming = useCallback(async () => {
+    const response = await fetch(
+      `https://cloud.fishjam.work/api/v1/connect/*YOUR_ID*/room-manager/${roomName}/users/${userName}`,
+    );
+    const { url, peerToken } = await response.json();
+
+    await prepareCamera({ cameraEnabled: true });
+
+    await joinRoom(url, peerToken);
+  }, []);
+
+  return <Button title={"Start Streaming"} onPress={startStreaming} />;
+}
+
+export default function HomeScreen() {
+  const [permission, requestPermission] = useCameraPermissions();
+
+  useEffect(() => {
+    requestPermission();
+  }, []);
+
+  if (!permission) {
+    return <View />;
+  }
+
+  return (
+    <SafeAreaView style={styles.container}>
+      <StartStreamingButton roomName={"*roomName*"} userName={"*username*"} />
+      <TracksView />
+    </SafeAreaView>
+  );
+}
+
+const styles = StyleSheet.create({
+  container: {
+    flex: 1,
+  },
+  tracksContainer: {
+    flex: 1,
+    alignItems: "center",
+  },
+  videoElement: {
+    width: "85%",
+    aspectRatio: 1,
+    backgroundColor: "gray",
+    marginVertical: 16,
+  },
+});
```

Comment:
I would prefer to avoid using styles in example code. Whenever I use example, I prefer to copy part of code and styles make it more difficult

Reply:
This must stay, or at least dimension style of `VideoRenderer`, otherwise they will not have size and nothing would  display.

Reply:
It's all in one snippet that you can copy, otherwise those snippets are useless and won't work I think this is the main goal of a 5 minute setup so that everything works.